tests/vmi_cloudinit_test.go

```diff
@@ -132,14 +132,25 @@ var _ = Describe("[rfe_id:151][crit:high][vendor:cnv-qe@redhat.com][level:compon
 	Describe("[rfe_id:151][crit:medium][vendor:cnv-qe@redhat.com][level:component]A new VirtualMachineInstance", func() {
 		Context("with cloudInitNoCloud userDataBase64 source", func() {
 			It("[test_id:1615]should have cloud-init data", func() {
-				userData := fmt.Sprintf("#!/bin/sh\n\necho '%s'\n", expectedUserData)
+				userData := fmt.Sprintf("#!/bin/sh\n\ntouch /%s\n", expectedUserData)
 
 				vmi := tests.NewRandomVMIWithEphemeralDiskAndUserdata(cd.ContainerDiskFor(cd.ContainerDiskCirros), userData)
-				LaunchVMI(vmi)
+
+				By("Run the VMI")
```

Comment:
This text adds a lot of overhead and I am not clear how it helps, even when debugging.
IMO it is overused, beyond its original purpose.

Funny thing is, that there is already such logging in `RunVMIAndExpectLaunch` itself, but that is not really my point here.

Reply:
In general it should help someone that is not familiar with the code to write down the test steps. But I didn't notice the helper functions include steps, I've removed them.

Reply:
The test steps should be clear from the code itself, using good function names.
Comments and logging text tends to rot and be irrelevant.

Reply:
@EdDev I meant when someone wants to document the steps in systems like HP QC or Polarion

Reply:
@EdDev Yes I agree. `By` should be used for complex specs.